test/models/patient_notification_eligibility_test.rb

```diff
@@ -520,11 +520,21 @@ def setup
     super
     Timecop.freeze(Time.parse('2021-03-14T18:00:00Z'))
   end
+
+  def teardown
+    super
+    Timecop.return
+  end
 end
 
 class PatientNotificationEligibilityTestWhenDSTEnds < PatientNotificationEligibilityTest
   def setup
     super
     Timecop.freeze(Time.parse('2021-11-07T18:00:00Z'))
   end
+
+  def teardown
+    super
+    Timecop.return
```

Comment:
I already looked at this in the other DST fixes we worked on. `super` already calls `Timecop.return` and this class also inherits the `teardown` method from `PatientNotificationEligibilityTest` so not sure if the entire method is required at all...

Unless you've tested and seen differently.

Reply:
Added for sanity check but if you verified otherwise then I can remove

Reply:
pushed the change